zerver/webhooks/github/view.py

```diff
@@ -910,13 +911,24 @@ def api_github_webhook(
     payload: JsonBodyPayload[WildValue],
     branches: str | None = None,
     user_specified_topic: OptionalUserSpecifiedTopicStr = None,
+    skip_private_repositories: Json[bool] = False,
 ) -> HttpResponse:
     """
     GitHub sends the event as an HTTP header.  We have our
     own Zulip-specific concept of an event that often maps
     directly to the X-GitHub-Event header's event, but we sometimes
     refine it based on the payload.
     """
```

Comment:
This seems like it was supposed to be a comment on the line below that assigns `header_event`, which you’ve now split it from, rather than a docstring for the function.